testsuite/cli/src/libra_client.rs

```diff
@@ -252,21 +251,11 @@ impl LibraClient {
                 new_state,
                 latest_epoch_change_li,
             } => {
-                info!(
-                    "Verified epoch changed to {}",
-                    latest_epoch_change_li
-                        .ledger_info()
-                        .next_epoch_state()
-                        .expect("no validator set in epoch change ledger info"),
-                );
                 // Update client state
                 self.trusted_state = new_state;
                 self.latest_epoch_change_li = Some(latest_epoch_change_li.clone());
             }
             TrustedStateChange::Version { new_state } => {
-                if self.trusted_state.latest_version() < new_state.latest_version() {
```

Comment:
I can see how this can be considered too 'noisy', but for me personally, I found this actually gives a good sanity check on a node's progress during a blocking call to a client request 😮